### PR TITLE
Parse Group Use Statements

### PR DESCRIPTION
**Pull request by ragboyjr**
- Updated namespace parser to handle grouped namespaces

Signed-off-by: RJ Garcia <rj@bighead.net>

### CODE REVIEW

**Comment by jaapio** [comment]
Thanks, this looks good. I will have a look later this week to see if we can merge and release this.

File: tests/unit/Types/ContextFactoryTest.php [hunk 1]
```diff
@@ -14,11 +14,14 @@
 
 // Added imports on purpose as mock for the unit tests, please do not remove.
     use \ReflectionClass;
```
Comment (ashnazg) [inline tests/unit/Types/ContextFactoryTest.php]: Can you move this line to come right after the PHPUnit grouping?  That way, the parsing rules will have a line after the grouping to process.  I'd like to see how the test goes after that change.

**Comment by jaapio** [comment]
I checked the change requested by @ashnazg. This is merged to master now.

**Comment by natebrunette** [comment]
@jaapio No rush, just curious when we can expect a release.

**Comment by jaapio** [comment]
Think that will be done this week. My day is almost ending, and I have no clue yet when I'm able to finish this merge. There are a number of code style checks failing. That I need to fix before the release

**Comment by jaapio** [comment]
Ok, time to stop for to day. 
But the release is there https://github.com/phpDocumentor/TypeResolver/releases/tag/0.7.0

Thanks @ragboyjr for adding this feature.

**Pull request merged**